src/web/components/SharingPermission/ParticipantsTable.tsx

```diff
@@ -112,7 +127,11 @@ function ParticipantsTableContent({
       <tbody>
         {sortedData.map((participant) => (
           <ParticipantItem
-            addedBy={showAddedByColumn ? 'Manual' : undefined} // TODO: Update this once we have auto add functionality
+            addedBy={
+              showAddedByColumn
+                ? formatStringsWithSeparator((participant as SharingParticipant).addedBy)
```

Comment:
Nit: I think the formatting should be done in the item instead - that way the checkbox can look for the existence of 'Manually Added' in the array, instead of searching the string for the substring 'Manually Added'. It feels like a cleaner approach.

Reply:
I think this is a really good idea. I've now moved formatStringsWithSeparator into the participant item, so that I could remove `showAddedBycolumn` from the participant table and make the type check much easier. 
I've also moved the `isAddedByManual` check into a helper to avoid string comparison in the component